### ISSUE TITLE
Only Windows clients unable to connect to server.

### ISSUE THREAD

**Issue by nhathaway** [jellyfin/jellyfin-media-player#684]
I updated to the latest jellyfin windows media player today and now it no longer connects. You can connect via the web browser but no longer via the jellyfin windows media player. It connected OK before the update. The new version is 1.11.0

**Comment by nhathaway** [comment]
I don't know if it makes any difference, but I access jellyfin over https using a reverse proxy.

Jellyfin server version: 10.9.6

**Comment by kloon15** [comment]
Same issue, suddenly wont connect at all, and the button stays disabled so cant even retry.

**Comment by nhathaway** [comment]
> Same issue, suddenly wont connect at all, and the button stays disabled so cant even retry.

Yes, I found that too. Once it had tried once, you can't retry without closing the app and restarting it again.

**Comment by vincejv** [comment]
I think we should merge it with https://github.com/jellyfin/jellyfin-media-player/issues/683

I'm experiencing the same with Linux Desktop client, all the mobile clients and chrome (web) works

**Comment by oddstr13** [comment]
Button staying disabled seems to happen when hitting the public api endpoint succeeds, but navigating to the web interface fails

**Comment by xAphex** [comment]
issue still persists with JMP 1.11.1 and jellyfin 10.9.6

**Comment by fatexs** [comment]
can you try deleting C:\Users\warpe\AppData\Local\Jellyfin Media Player and C:\Users\warpe\AppData\Local\JellyfinMediaPlayer ?

**Comment by xAphex** [comment]
> can you try deleting C:\Users\user\AppData\Local\Jellyfin Media Player and C:\Users\user\AppData\Local\JellyfinMediaPlayer ?

already did that and uninstalled JMP, its a problem with JMP, it reaches an endpoint that requires authentication and stops there.

**Comment by IradColdheart** [comment]
Just in case, i faced the same issue : it was working through the reverse proxy, but through a direct access. 
Adding the subpath to the IP for direct access corrected the issue though : http://192.168.1.x:8096/subpath (url subpath being defined in Jellyfin network settings).

**Comment by valankar** [comment]
How come this is still not fixed? I still have to resort to older version.

**Comment by xAphex** [comment]
> How come this is still not fixed? I still have to resort to older version.

im at a point where im waiting for https://github.com/prayag17/Blink to mature a bit more then i will ditch JMP, the maintainer doesn't care about fixing the mess they started, i even mentioned the problem to them personally on discord and they never replied, the problem is: when connecting to a server the connection reaches an authentication endpoint and stops there.

**Comment by timelord2018** [comment]
Find and open C:\Users\*yourname*\AppData\Local\JellyfinMediaPlayer\jellyfinmediaplayer.conf
Find  file "userWebClient": and revise your link . e.g.: http*******.
And restart app

**Comment by vincejv** [comment]
@valankar i can't repro this on my Windows machine, that's probably why it's not fixed, it's not easily reproducible, in order to fix a bug the dev should be able to repro in his/her environment

@timelord2018 if that's the fix for the issue the installer/uninstaller should have an option to clear app data when uninstalling Jellyfin from Windows machines so an uninstall/reinstall will fix these issues when upgrading to a newer version

**Comment by nhathaway** [comment]
> Find and open C:\Users*yourname*\AppData\Local\JellyfinMediaPlayer\jellyfinmediaplayer.conf Find file "userWebClient": and revise your link . e.g.: http*******. And restart app

This works for me. It was set to "", and setting it to my URL fixed the problem. But this is not the kind of fix the average user would be in a position to make.

**Comment by chaddy314** [comment]
> >Find and open C:\Usersyourname\AppData\Local\JellyfinMediaPlayer\jellyfinmediaplayer.conf Find file "userWebClient": and revise your link . e.g.: http*******. And restart app

>This works for me. It was set to "", and setting it to my URL fixed the problem. But this is not the kind of fix the average user would be in a position to make.

This doesn't seem to work on flatpak client (with `~/.var/app/com.github.iwalton3.jellyfin-media-player/data/jellyfinmediaplayer/jellyfinmediaplayer.conf`). 
Server Version: 10.9.9
Client Version: 1.11.1

**Comment by CMHarris14** [comment]
I have been having this same issue with 1.11 and have just stuck to 1.10 for now. On a whim I decided to completely remove jellyfin from my pc and reinstall with winget rather than download it here. 1.11 connects perfectly fine now. I can't fathom why that made a difference, but I wanted to mention it here in case that helps anyone.

**Comment by xAphex** [comment]
> I have been having this same issue with 1.11 and have just stuck to 1.10 for now. On a whim I decided to completely remove jellyfin from my pc and reinstall with winget rather than download it here. 1.11 connects perfectly fine now. I can't fathom why that made a difference, but I wanted to mention it here in case that helps anyone.

#755 fixed the issue for me

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
